parser/parser.y

```diff
@@ -1009,36 +1017,40 @@ AlterTableSpec:
 	{
 		$$ = &ast.AlterTableSpec{
 			Tp:		ast.AlterTableModifyColumn,
-			NewColumn:	$3.(*ast.ColumnDef),
-			Position:	$4.(*ast.ColumnPosition),
+			NewColumns:	[]*ast.ColumnDef{$3.(*ast.ColumnDef)},
+            Positions:	[]*ast.ColumnPosition{$4.(*ast.ColumnPosition)},
```

Comment:
ditto.
PS, if you use vim, you should `set tabstop=8` to avoid this problem. Other editors should have similar settings.